sql/src/main/antlr/OpenSearchSQLIdentifierParser.g4

```diff
@@ -63,4 +63,6 @@ keywordsCanBeId
     | COUNT | SUM | AVG | MAX | MIN
     | TIMESTAMP | DATE | TIME | DAYOFWEEK
     | FIRST | LAST
+    | CURRENT_DATE | CURRENT_TIME | CURRENT_TIMESTAMP | LOCALTIME | LOCALTIMESTAMP | UTC_TIMESTAMP | UTC_DATE | UTC_TIME
+    | CURDATE | CURTIME | NOW
```

Comment:
wondering if any special reason not to add `constantFunctionName` here or in `ident` rule directly?

Reply:
I may have missed the circular reference between identifier.g4 and parser.g4..